sinks.go

```diff
@@ -42,10 +42,13 @@ type Sink interface {
 	view() (ByteView, error)
 }
 
+// I have to change this code
```

Comment:
If the cache content is very large, this is going to be very slow. After changed this code, nothing seems going wrong.